source/app/service-providers/fsal/index.ts

```diff
@@ -140,6 +140,40 @@ export default class FSAL extends ProviderContract {
       this._logger.info('Clearing the FSAL cache ...')
       this.clearCache()
     }
+
+    // Start a timer to measure how long the roots take to load.
+    const start = Date.now()
```

Comment:
Date.now is not exact enough for perf measurements. Something like https://nodejs.org/api/perf_hooks.html#performancenow is preferred (not sure though if it works in electron)

Reply:
It's not really a performance measure. It should rather serve as a rule of thumb indication which basically only I use during development to have a coarse measure of how well the FSAL does (so I see it in case I accidentally break something)

Reply:
Has worked, so why not, we're now using the `performance` hook!